src/validation.cpp

```diff
@@ -2929,6 +2931,7 @@ bool Chainstate::FlushStateToDisk(
         }
         // Flush best chain related state. This can only be done if the blocks / block index write was also done.
         if (fDoFullFlush && !CoinsTip().GetBestBlock().IsNull()) {
+            if (coins_mem_usage >= WARN_FLUSH_COINS_SIZE) LogWarning("Flushing large %d GiB UTXO set to disk...", coins_mem_usage >> 30);
```

Comment:
Thanks for the constant.

This is now printing GiB, and the very next log line will be printing kB. Not exactly pretty, but I suppose it's only when bench is on.

Also, to the average user seeing this, it may not be exactly clear what you're trying to communicate. How about adding "This may take some time." or so?

Reply:
> This is now printing GiB, and the very next log line will be printing kB. Not exactly pretty, but I suppose it's only when bench is on.

Yeah, I saw that, it's dividing by 1000 - changed to `KiB` and `>> 10`.